gabriel-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -74,21 +74,56 @@ <h2 class="h2-index">Check Out some pictures I've taken!</h2>
       </a>
 
       <h2 class="h2-index">Comments</h2>
-
+      
+      <!-- Form to request comments from the database -->
       <form action="javascript:getComments();">
-        <label for="maxNumberOfComments">Enter the number of comments to wish to see:</label>
-        <input type="number" id="maxNumberOfComments" name="maxNumberOfComments" min="1" max="1000">
+        <div class="comment-divs">
+          <label for="language">Choose a language for the comments:</label>
+          <select name="language" class="select-language">
+            <option value="en">English</option>
+            <option value="es">Spanish</option>
+            <option value="zh">Chinese</option>
+            <option value="ar">Arabic</option>
+            <option value="hi">Hindi</option>
+          </select>
+        </div>
+
+        <div id="max-comments">
+          <label for="maxComments">Enter the number of comments to wish to see:</label>
+          <input type="number" name="maxComments" id="enter-max-comments" min="1">
+        </div>
+
         <input type="submit">
       </form>
 
       <div id="comments-container"></div>
 
       <br>
-
+      
+      <!-- Form to submit a comment to the database -->
       <form action="/data" method="POST">
-        <label for="commentBox">enter a new comment</p>
-        <textarea id="commentBox" name="text-input">type here</textarea>
-        <br>
+        <div class="comment-divs">
+          <label for="languageBox">Choose a language for your comment:</label>
```

Comment:
I believe google translate can auto detect language from text.

Reply:
I am taking it so people will be skeptical about translated comments. Sometimes a comment that was written in Spanish might be lost in translation.